mautic/mautic
# Enable video attributes in Froala

Please answer the following questions. 

| Q | A |
| --- | --- |
| Bug fix? | Y |
| New feature? | N |
| Related user documentation PR URL | / |
| Related developer documentation PR URL | / |
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/pull/2032#issuecomment-238078698 |
| BC breaks? | N |
| Deprecations? | N |
#### Description:

Some video HTML attributes get stripped by Froala.
#### Steps to test this PR:
1. Apply this PR
2. Refresh production assets
3. Refresh browser 
4. Repeat the test.
5. The attributes should stay there
#### Steps to reproduce the bug:
1. Try to save this as DWC content:

```
<div class="videoWrapper">
    <iframe src="https://player.vimeo.com/video/XXXXXXXXX?byline=0&amp;portrait=0&amp;badge=0" width="100%" height="100%" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
</div>
```

Notice that `frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen` attributes are stripped.
